### PR TITLE
[JW8-9168] Polyfill openLink for Firefox

### PR DESCRIPTION
**Pull request by zetagame**
### This PR will...
Replace the .click() functionality of openLink with a new click mouse event.

### Why is this Pull Request needed?
Firefox doesn't support default click event on anchors unless they're appended to the DOM. Using the mouse event constructor allows us to avoid doing this and still have full functionality.

#### Addresses Issue(s):

JW8-9178

### CODE REVIEW

**Comment by johnBartos** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  🛠 There are modified src files, but no test changes. Add tests if you're able to.
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  🗿 Set a milestone. It should be the ticket's fix version in JIRA.
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by jnatalzia** [comment]
test this please

File: src/js/utils/dom.js [hunk 1]
```diff
@@ -215,5 +215,5 @@ export function openLink(link, target, additionalOptions = {}) {
     a.href = link;
     a.target = target;
     a = Object.assign(a, additionalOptions);
-    a.click();
+    a.dispatchEvent(new MouseEvent(`click`, { bubbles: true, cancelable: true, view: window }));
```
Comment (robwalch) [inline src/js/utils/dom.js]: Has this been tested on IE11?

Comment (zetagame) [reply src/js/utils/dom.js]: On browser stack IE11, yes.

**Pull request merged**